esmvaltool/cmorizers/data/formatters/datasets/piomas.py

```diff
@@ -21,17 +21,17 @@
    should be achievable for most of them just modifying the config file
 """
 
+import collections
+import glob
 import logging
 import os
-import glob
-import collections
 
+import iris
 import numpy as np
 from cf_units import Unit
-import iris
 from iris.coords import AuxCoord, DimCoord
 
-from .utilities import save_variable, set_global_atts
+from esmvaltool.cmorizers.data.utilities import save_variable, set_global_atts
 
 logger = logging.getLogger(__name__)
 
```

Comment:
This script is currently not working because of the call to the cmorization function.

Reply:
Fixed